src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Ricci.hpp

```diff
@@ -107,6 +107,7 @@ namespace GeneralizedHarmonic {
  *          + \frac{1}{4} \Phi_{ik}{}^l \Phi_{jl}{}^k
  *          + \frac{1}{2} \left(\Phi^k{}_{il} \Phi_{kj}{}^l
  *                              - \Phi^k{}_{li} \Phi^l{}_{kj}\right).
+ * \label{eq:rij}
```

Comment:
I moved this down here so that the link to eq [64] is properly linked (see current broken link at the bottom of the documentation for this function [here](https://spectre-code.org/group__GeneralRelativityGroup.html#ga72db980d7ce1a591bdff48cd353d000d)).

Separately, note that in the rendered documentation for this PR for the `void` version of this `spatial_ricci_tensor` function, the highlighted yellow error message that reads **Label 'eq:rij' multiply defined** is pre-existing (see the error message in current documentation [here](https://spectre-code.org/group__GeneralRelativityGroup.html#ga582aa60e616b2257b6d28808f704c64f))